browser/scripts/background.js

```diff
@@ -103,6 +103,15 @@ function handleMessages({code, count}, {tab}) {
 			text,
 			tabId: tab.id
 		});
+	} else if (code === 'focus-tab') {
+		// We need to query it again because windowId might have changed in the
+		// meanwhile.
+		chrome.tabs.get(tab.id, tab => {
+			// Focus window.
+			chrome.windows.update(tab.windowId, {focused: true});
+			// Focus tab.
+			chrome.tabs.update(tab.id, {active: true});
+		});
```

Comment:
`chrome.runtime.sendMessage` is handled right away and the data is fresh, there's no way it can have changed in this case. Also the code is self-explanatory
```suggestion
		chrome.tabs.update(tab.id, {active: true});
		chrome.windows.update(tab.windowId, {focused: true});
```